ui/app/templates/vault/cluster/secrets/backend/error.hbs

```diff
@@ -33,33 +33,19 @@
         and navigating from there.
       </p>
     {{else if (eq this.model.httpStatus 403)}}
-      {{#if (eq this.model.policyPath "sys/capabilities-self")}}
-        <p>
-          Your auth token does not have access to
-          <code>{{this.model.policyPath}}</code>. Vault Enterprise uses this endpoint to determine what actions are allowed
```

Comment:
Is this necessarily true? I think we have this endpoint too: https://openbao.org/api-docs/system/capabilities-self/